.github/workflows/cd_wiki.yaml

```
@@ -0,0 +1,46 @@
+name: Wiki CD
+
+on:
+  workflow_dispatch:
+
+concurrency:
+  group: 'wiki-cd'
+
+jobs:
+
+  cd_wiki:
+    name: Wiki deployment
+    runs-on: ubuntu-20.04
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout endless-sky-wiki repository
+        run: |
+          git clone "https://github.com/endless-sky/endless-sky-wiki.git"
+      - name: Checkout wiki
+        run: git clone "https://github.com/${{ github.repository }}.wiki.git"
+      - name: Synchronise files
+        run: rsync --delete -rvh -e .git endless-sky-wiki/wiki/* endless-sky.wiki/
+      - name: Check for changes
+        id: check-diff
+        run: |
+          cd endless-sky.wiki
+          git add .
+          if [[ $(git diff --cached --raw) ]]; then
+            echo "changes=yes" >> "$GITHUB_OUTPUT"
+          fi
+      - name: Commit changes
+        if: ${{ steps.check-diff.outputs.changes == 'yes' }}
+        run: |
+          cd endless-sky.wiki
+          git config --local user.name "GitHub Actions"
+          git config --local user.email "actions@github.com"
+          git commit -m "Update" -m "Triggered by ${{ github.actor }}"
```

```suggestion
          git commit -m "Synchronize with endless-sky-wiki" -m "Triggered by ${{ github.actor }}"
```

Is there an easy way to get the commit range that is being pushed to the wiki? It would be neat if that could be added as extra lines to the commit message. (Either just the commit range with links to the wiki repo, or each commit on a separate line?)

---

I don't know how that would be done, but I can think of some things that might work. I'll look into it.